backend/review/documentation.py

```diff
@@ -132,6 +132,10 @@
                         "type": "string",
                         "description": "The dash-joined department and most-recent code of this course, e.g. `CIS-1200`.",  # noqa E501
                     },
+                    "superseded_last_offered": {
```

Comment:
This field is kind of confusing. Is there anything to do about the naming here? I'm coming up blank right now.